src/transformers/utils/doc.py

```diff
@@ -242,7 +242,7 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
     >>> num_labels = len(model.config.id2label)
     >>> model = {model_class}.from_pretrained("{checkpoint}", num_labels=num_labels)
 
-    >>> labels = torch.tensor(1)
+    >>> labels = torch.tensor([1])
```

Comment:
I modified this line because it threw an error regarding the dimensions of the tensor:
`IndexError: dimension specified as -1 but tensor has no dimensions `

But now it works well (I also tested for RoBerta, to make sure it works the same)